app/src/test/java/com/codesoom/assignment/controllers/TaskControllerTest.java

```diff
@@ -0,0 +1,163 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskController 를 테스트 한다.")
+class TaskControllerTest {
+    private static final String TASK_TITLE_1 = "title1";
+    private static final String TASK_TITLE_2 = "title2";
+    private Task task1;
+    private Task task2;
+    private List<Long> givenInvalidIds;
+
+    private TaskController taskController;
+
+    @BeforeEach
+    void setUp() {
+        TaskService taskService = new TaskService();
+        taskController = new TaskController(taskService);
+
+        task1 = new Task();
+        task1.setTitle(TASK_TITLE_1);
+
+        task2 = new Task();
+        task2.setTitle(TASK_TITLE_2);
+
+        givenInvalidIds = new ArrayList<>();
+        Collections.addAll(givenInvalidIds, -100L, -1L, 0L, 5L, 100L);
+    }
+
+    @Test
+    @DisplayName("할 일이 없는 경우 할 일 목록을 가져오면, 비어있는 리스트가 리턴된다.")
```

Comment:
콤마를 사용해 실험 조건과 실험 과정, 실험 결과를 분리해보면 더 읽기 좋을 것 같습니다.
```suggestion
    @DisplayName("할 일이 없는 경우, 할 일 목록을 가져오면, 비어있는 리스트가 리턴된다.")
```